Kitodo/src/main/java/org/kitodo/production/ldap/LdapUser.java

```diff
@@ -292,7 +292,7 @@ public Attributes getAttributes(Name name) throws NamingException {
 
     @Override
     public Attributes getAttributes(String name, String[] ids) throws NamingException {
-        if (StringUtils.isBlank(name)) {
+        if (!StringUtils.isBlank(name)) {
```

Comment:
```suggestion
        if (StringUtils.isNotBlank(name)) {
```
Please use the dedicated, negated check instead of manually negating the positive check here, as it is more readable.